src/main.js

```diff
@@ -51,43 +52,57 @@ bot.on('message', (message) => {
     switch(args[0]){ 
       // Faucet commands
       case 'goerliEth': {
+        let embed = new Discord.MessageEmbed().setColor(3447003).setTimestamp();
         if (args[1] == null) {
-          message.channel.send('Goerli ETH adress required.')
+          embed.setDescription('Goerli ETH address is required.');
+          message.lineReply(embed);
+//           message.channel.send({embed});
           break;
         } else if (!web3.utils.isAddress(args[1])) {
```

Comment:
It might be the place where address is properly checked to not be malicious. Should check  `web3.utils.isAddress` method to be sure.

Reply:
Yes as soon as in this place address check works, the SQL queries are safe, but for the rest of params used in SQLs there should be exact types.

From `isAddress` method: `/^(0x)?[0-9a-f]{40}$/i`